master/getting-started/docker/index.md

```diff
@@ -32,15 +32,20 @@ you need to install and use Calico.
 
 ## Tutorials
 
-#### [Simple Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-policy)
+#### [Simple Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-profile-policy)
 
 The above guide demonstrates Calico connectivity cross host, and how to limit
 that connectivity using simple Calico policy.
 
-#### [Advanced Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-policy)
+#### [Advanced Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-profile-policy)
 
 The above guide digs deeper into advanced policy configurations for workloads.
 
+#### [Advanced Label Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-label-policy)
+
+The above guide demonstrates Calico connectivity cross host, and how to limit
```

Comment:
Whereas here, we will be using the container-specific labels to apply Calico label-based policy.  This allows policy tweaks at the container level rather than at the network level.